Integrations/SignalSciences/SignalSciences.py

```diff
@@ -0,0 +1,1001 @@
+        'Contents': request_response_data,
+        'ContentsFormat': formats['json'],
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': human_readable,
+        'EntryContext': {
+            'SigSciences.CorpLists(val.ID==obj.ID)': entry_context,
+        }
+    })
+
+
+def get_all_corp_lists():
+    url = SERVER_URL + CREATE_CORP_LIST_SUFFIX.format(CORPNAME)
+    response_data = http_request('GET', url)
+    return response_data
+
+
+def get_all_corp_lists_command():
+    request_response_data = get_all_corp_lists()
+    list_of_corp_lists = request_response_data.get('data', [])
+
+    corp_lists_contexts = []
+    for corp_list_data in list_of_corp_lists:
+        cur_corp_list_context = list_entry_context_from_response(corp_list_data)
+        corp_lists_contexts.append(cur_corp_list_context)
+    sidedata = "Number of corp lists in corp: {0}".format(len(list_of_corp_lists))
+
+    demisto.results({
+        'Type': entryTypes['note'],
+        'Contents': request_response_data,
+        'ContentsFormat': formats['json'],
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': tableToMarkdown(LIST_OF_CORP_LISTS_TITLE, corp_lists_contexts, metadata=sidedata),
+        'EntryContext': {
+            'SigSciences.CorpLists(val.ID==obj.ID)': corp_lists_contexts,
+        }
+    })
+
+
+def create_site_list(siteName, list_name, list_type, entries_list, description=None):
+    if not is_legal_list_type(list_type):
+        return_error("Error: {0} is not a legal type for a list. Legal types are IP, String, "
+                     "Country or Wildcard".format(list_type))
+    url = SERVER_URL + SITE_CREATE_LIST_SUFFIX.format(CORPNAME, siteName)
+    entries_list_in_list_format = entries_list.split(',')
+    data_for_request = {
+        'name': list_name.lower(),
+        'type': list_type.lower(),
+        'entries': entries_list_in_list_format
+    }
+    if description is not None:
+        data_for_request['description'] = description
+    new_list_data = http_request('POST', url, data=data_for_request)
+    return new_list_data
+
+
+def create_site_list_command():
+    args = demisto.args()
+    request_response_data = create_site_list(args['siteName'], args['list_name'],
+                                             args['list_type'], args['entries_list'], args.get('description', None))
+    entry_context = list_entry_context_from_response(request_response_data)
+    title = "Signal Sciences - creating a new site list \n\n The list has been succesfully created"
+    human_readable = tableToMarkdown(title, entry_context)
+    demisto.results({
+        'Type': entryTypes['note'],
+        'Contents': request_response_data,
+        'ContentsFormat': formats['json'],
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': human_readable,
+        'EntryContext': {
+            'SigSciences.SiteLists(val.ID==obj.ID)': entry_context,
```

Comment:
You don't have ID in the context, please verify that you check an existing field, here and everywhere